app/src/main/java/com/keystone/cold/util/URRegistryHelper.java

```diff
@@ -0,0 +1,99 @@
+package com.keystone.cold.util;
+
+import com.keystone.coinlib.utils.B58;
+import com.keystone.cold.callables.GetExtendedPublicKeyCallable;
+import com.keystone.cold.callables.GetMasterFingerprintCallable;
+import com.sparrowwallet.hummingbird.registry.CryptoCoinInfo;
+import com.sparrowwallet.hummingbird.registry.CryptoHDKey;
+import com.sparrowwallet.hummingbird.registry.CryptoKeypath;
+import com.sparrowwallet.hummingbird.registry.PathComponent;
+
+import org.spongycastle.util.encoders.Hex;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+
+public class URRegistryHelper {
+    public static CryptoHDKey generateETHCryptoHDKey() {
+        byte[] masterFingerprint = Hex.decode(new GetMasterFingerprintCallable().call());
+        String xPub = new GetExtendedPublicKeyCallable("m/44'/60'/0'").call();
+        ExtendedPublicKey extendedPublicKey = new ExtendedPublicKey(xPub);
+        List<PathComponent> pathComponents = Arrays.asList(new PathComponent(44, true),
+                new PathComponent(60, true),
+                new PathComponent(0, true));
+        CryptoKeypath origin = new CryptoKeypath(pathComponents, masterFingerprint, (int) extendedPublicKey.getDepth());
+        return new CryptoHDKey(false, extendedPublicKey.getKey(),
+                extendedPublicKey.getChainCode(),
+                new CryptoCoinInfo(0, 0),
```

Comment:
Modify this line, refer to: https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-007-hdkey.md#cddl-for-coin-info